### ISSUE TITLE
[Feature request] BridgedDeviceBasicInformation cluster

### ISSUE THREAD

**Issue by lboue** [SiliconLabs/arduino#13]
Hi,

Can you add [BridgedDeviceBasicInformation ](https://github.com/project-chip/connectedhomeip/blob/master/docs/clusters.md)cluster to your examples?
This makes it possible to present a comprehensible device name in the user interface.

## List of currently defined clusters

| Code (dec) | Code (hex) | Name                                                    |
| ---------- | ---------- | ------------------------------------------------------- |
|          3 |       0x03 | Identify                                                |
|          4 |       0x04 | Groups                                                  |
|         .. |       ..   | ...				                                        |
|         **57** |       **0x39** | **BridgedDeviceBasicInformation**                           |
|         59 |       0x3B | Switch                                                  |

### BridgedDeviceBasicInformation
Here's an example for this cluster:

```json
        "7/57/5": "Hue dimmer salon",
        "7/57/17": true,
        "7/57/2": 4107,
        "7/57/1": "Signify Netherlands B.V.",
        "7/57/3": "RWL021",
        "7/57/10": "1.1.28573",
        "7/57/18": "115dc69142c249e1a217447262f98823",
        "7/57/14": "Hue dimmer switch",
        "7/57/65532": 0,
        "7/57/65533": 1,
        "7/57/65528": [],
        "7/57/65529": [],
        "7/57/65531": [
          5,
          17,
          2,
          1,
          3,
          10,
          18,
          14,
          65528,
          65529,
          65531,
          65532,
          65533
        ]
```

**Comment by silabs-bozont** [comment]
Hello lboue,
Thank you, great suggestion! This is coming in the next version along with functions for customizing your devices, like setting it's name/manufacturer/serial.

**Comment by lboue** [comment]
> Hello lboue, Thank you, great suggestion! This is coming in the next version along with functions for customizing your devices, like setting it's name/manufacturer/serial.

Great!

**Comment by silabs-bozont** [comment]
Here's a little sneak peek:
![IMG_0028](https://github.com/SiliconLabs/arduino/assets/124884281/833f1b12-305d-43d5-998c-f6524269cd33)

**Comment by lboue** [comment]
> You'll be able to set these values to anything you'd like.
> 
> 

It's perfect!

**Comment by silabs-bozont** [comment]
It's now available in 2.0.0
